WWDCFamily/Library/Models/Session.swift

```diff
@@ -1,13 +1,14 @@
 import Foundation
+import Firebase
 
 struct Session {
```

Comment:
It would be helpful to have a sharedInstance that keeps track of the current users session.

This class could also house calls to another object that handles authentication